### PR TITLE
Fix for network.AggregateInterface path issues

### PR DESCRIPTION
**Pull request by patrickdaj**
* Fixed incorrect paths in network.AggregateInterface based more or less on network.EthernetInterface.
* Added enable_lacp option to turn on LACP.
* Added lacp_passive_pre_negotiation to enable Passive Pre-Negotiation.
* Added conditions based on what was in network.EthernetInterface.

### CODE REVIEW

File: pandevice/network.py [hunk 1]
```diff
@@ -1099,30 +1099,29 @@ class AggregateInterface(PhysicalInterface):
...
-        ipv4_mss_adjust(int): TCP MSS adjustment for ipv4
-        ipv6_mss_adjust(int): TCP MSS adjustment for ipv6
+        ipv4_mss_adjust(int): Layer3: TCP MSS adjustment for ipv4
+        ipv6_mss_adjust(int): Layer3: TCP MSS adjustment for ipv6
         enable_dhcp (bool): Enable DHCP on this interface
-        create_dhcp_default_route (bool): Create default route pointing to default gateway provided by server
-        dhcp_default_route_metric (int): Metric for the DHCP default route
+        create_dhcp_default_route (bool): Layer3: Create default route pointing to default gateway provided by server
+        dhcp_default_route_metric (int): Layer3: Metric for the DHCP default route
+        enable_lacp (bool): Enables LACP
```
Comment (shinmog) [inline pandevice/network.py]: Change this to `lacp_enable` to match what the code below is naming the variable.

File: pandevice/network.py [hunk 2]
```diff
@@ -1196,7 +1202,22 @@ def _setup(self):
...
+            condition={'mode': ['layer3', 'layer2', 'ha'], 'lacp_enable': True},
+            vartype='yesno', path='{mode}/lacp/passive-pre-negotiation'))
+        params.append(VersionedParamPath(
+            'lacp_mode',
+            condition={'mode': ['layer3', 'layer2', 'ha'], 'lacp_enable': True},
+            path='{mode}/lacp/mode'))
+        params.append(VersionedParamPath(
+            'lacp_rate',
+            condition={'mode': ['layer3', 'layer2', 'ha'], 'lacp_enable': True},
+            path='{mode}/lacp/transmission-rate'))
```
Comment (shinmog) [inline pandevice/network.py]: The docstring mentions valid values for these two.  Add a `choices=[...]` to these two to assist users.

File: pandevice/network.py [hunk 3]
```diff
@@ -1196,7 +1202,22 @@ def _setup(self):
...
         self._params = tuple(params)
```
Comment (shinmog) [inline pandevice/network.py]: Need a blank line between params setup and the assignment.

**Comment by patrickdaj** [comment]
Resolved issues from review.

File: pandevice/network.py (hunk 2 above)
Comment (patrickdaj) [reply pandevice/network.py]: Assuming you meant values not choices, right?  I don't see choices in VersionedParamPath or ParamPath class.

**Pull request merged**